vaadin-spring/src/main/java/com/vaadin/flow/spring/security/VaadinWebSecurityConfigurerAdapter.java

```diff
@@ -108,4 +109,48 @@ public static RequestMatcher getDefaultWebSecurityIgnoreMatcher() {
                 .map(AntPathRequestMatcher::new).collect(Collectors.toList()));
     }
 
+    /**
+     * Sets up login for the application using form login with the given path
+     * for the login view.
+     * <p>
+     * This is used when your application uses a Fusion based login view
+     * available at the given path.
+     * 
+     * @param http
+     *            the http security from {@link #configure(HttpSecurity)}
+     * @param fusionLoginViewPath
+     *            the path to the login view
+     * @throws Exception
+     *             if something goes wrong
+     */
+    protected void setLoginView(HttpSecurity http, String fusionLoginViewPath)
+            throws Exception {
+        setLoginView(http, fusionLoginViewPath, "/");
+    }
+
+    /**
+     * Sets up login for the application using form login with the given path
+     * for the login view.
+     * <p>
+     * This is used when your application uses a Fusion based login view
+     * available at the given path.
+     * 
+     * @param http
+     *            the http security from {@link #configure(HttpSecurity)}
+     * @param fusionLoginViewPath
+     *            the path to the login view
+     * @param logoutUrl
+     *            the URL to redirect the user to after logging out
+     * @throws Exception
+     *             if something goes wrong
+     */
+    protected void setLoginView(HttpSecurity http, String fusionLoginViewPath,
```

Comment:
is it specific to Fusion or apply to Flow as well? Better to rename it if it applies to both. Also, don't forget the javadoc when renaming.

Reply:
It is intended for Fusion. The Flow version in #780 takes a `Component` class instead of a String